### PR TITLE
completed code challenge

### PR DESCRIPTION
**Pull request by jquaglia**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: challenges/arrayShift/array-shift.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+function insertShiftArray(array, value){
+  const middle = array.length / 2;
+  const middleIndex = Math.ceil(middle);
+  array.splice(middleIndex, 0, value);
```
Comment (adrianhuebner) [inline challenges/arrayShift/array-shift.js:6]: I would love to see this done without .splice()
